### PR TITLE
chore: updated the readme file.

### PR DESCRIPTION
**Pull request by shubhw12**

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -5,6 +5,9 @@ Tested up to: 5.1
 Stable tag: 1.0.1
 Requires at least: 4.4
 
+>**Important:**</br>
+This plugin has been closed as of April 2, 2020 and is not available for download on WordPress plugins repository. This closure is permanent.
```
Comment (Nikschavan) [inline readme.txt:9]:
```suggestion
This plugin has been closed as of April 2, 2020 and is not available for download on WordPress plugins repository. This closure is permanent.

We recommend using [Astra Bulk Edit](https://wordpress.org/plugins/astra-bulk-edit/) instead. [This doc](link-to-doc) explains how to use it.
```

File: readme.txt [hunk 2]
```diff
@@ -5,6 +5,9 @@ Tested up to: 5.1
 Stable tag: 1.0.1
 Requires at least: 4.4
 
+>**Important:**</br>
```
Comment (Nikschavan) [inline readme.txt:8]: Is formatting here incorrect?

Comment (shubhw12) [reply readme.txt:8]: here > is used for blockquote and ** for making the important text bold.
This looks like this
https://share.getcloudapp.com/JruDkm12

Comment (Nikschavan) [reply readme.txt:8]: Actually we should remove everything from readme.md and just keep this message. readme.txt can have existing content and this extra string.

Comment (shubhw12) [reply readme.txt:8]: So shall I remove all the content and keep this message only?

Comment (Nikschavan) [reply readme.txt:8]: Yes - in readme.md

File: readme.txt (hunk 1 above)
Comment (shubhw12) [reply readme.txt:9]: Done with these changes

**Pull request merged**